testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/ContainerConstants.java

```diff
@@ -1,34 +1,38 @@
-package org.jboss.resteasy.test;

```

Comment:
Wrong line endings? https://issues.jboss.org/browse/RESTEASY-2115

Reply:
This is due to how the merge was done, I think.